crates/turborepo-lib/Cargo.toml

```diff
@@ -42,7 +42,7 @@ reqwest = { workspace = true, default_features = false, features = ["json"] }
 rustc_version_runtime = "0.2.1"
 semver = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
-serde_json = { workspace = true }
+serde_json = { version = "1.0.94", package = "serde_jsonc" }
```

Comment:
This is a floating, automated patch on top of `serde_json` for parsing JSONC. No API changes, release cadence is 1:1 (with up to a 1 hour delay because of cron polling).

I didn't elect to push it to everybody else without having a conversation first.